src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -354,7 +354,21 @@ public ResponseEntity<CallbackResponse> reprint(@RequestBody CallbackRequest cal
 
     @PostMapping(path = "/evidenceAdded", consumes = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<CallbackResponse> evidenceAdded(@RequestBody CallbackRequest callbackRequest) {
-        evidenceUploadService.updateLastEvidenceAddedDate(callbackRequest.getCaseDetails());
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        CaseData caseData = caseDetails.getData();
+        if (caseDetails.getState().equalsIgnoreCase("BOCaseStopped")) {
```

Comment:
I think we use STATE_BO_CASE_STOPPED from StateConstants

Reply:
When I was doing it locally, it was going to BOCaseStopped whenever I stopped the case, I will add an OR to it though

Reply:
Not sure what you are asking, we have a value defined for it in the code so you could use that instead of hardcoded in quotes? Not that it's likely to change tbh